### PR TITLE
Moved the metadata from setup.py into setup.cfg

### PR DESCRIPTION
**Pull request by KOLANICH**
>(Soon to be) the fastest pure-Python PEG parser 

In fact the fastest I have benchmarked.

### CODE REVIEW

**Comment by KOLANICH** [comment]
@erikrose

**Comment by evanh** [comment]
Can you clarify why you made this change?

**Comment by KOLANICH** [comment]
For better security when building packages somewhen in future.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools>=61.2.0", "wheel", "setuptools_scm[toml]>=3.4.3"]
```
Comment (lucaswiman) [inline pyproject.toml:2]: Does this mean the end user needs to have these tools installed or just the person who builds the dist file and wheel?

**Review by lucaswiman — CHANGES_REQUESTED** [review]
One small change. I was able to do an editable install (`pip install -e .`) and generate a sensible looking dist file. After some googling, I haven't found a straight answer about whether setup.cfg is preferred or pyproject.toml is. It seems like both are maintained?

Since you're suggesting this change, could you comment on why .toml is preferred over .cfg?

**Comment by KOLANICH** [comment]
1. about whether setup.cfg is preferred or pyproject.toml

`setup.cfg` is deprecated since PEP 621 being implemented in `setuptools`. 

2. >Does this mean the end user needs to have these tools installed or just the person who builds the dist file and wheel?

Only for the one building packages. `pyproject.toml` is used when python packages are built from source. Wheels uave own metadata.

**Review by lucaswiman — APPROVED** [review]
OK, LGTM. Thanks for this!

**Pull request merged**